firmware/models.py

```diff
@@ -0,0 +1,112 @@
+"""Mappers for firmware database tables"""
+
+from sqlalchemy import Column, ForeignKey, Integer, String
+from sqlalchemy.orm import relationship
+from firmware.database import Base, engine
+
+
+class User(Base):
+    """defines / maps users table in database"""
+    __tablename__ = 'users'
+
+    id = Column(Integer, primary_key=True)
+    username = Column(String(21), unique=True, nullable=False)
+    password = Column(String(100), nullable=False)
+    email = Column(String(50), unique=True, nullable=False)
+    name = Column(String(100))
+    surname = Column(String(100))
+    avatar = Column(String(100), nullable=False)
+    contact = Column(String(100))
+    privilege = Column(String(100), nullable=False)
+    gender = Column(String(100), nullable=False)
+    usernname_review = relationship("Review", backref='user',
```

Comment:
Please rename `usernname_review` to `reviews`.

This should be a list of reviews.